src/components/IdentitySettings.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { SimpleTextInput } from './SimpleTextInput';
+import {
+    KeyboardAvoidingView,
+    StyleSheet,
+    TouchableOpacity,
+    View,
+    Text,
+    Button,
+    Image,
+} from 'react-native';
+import { Author } from '../models/Post';
+import { AsyncImagePicker } from '../AsyncImagePicker';
+import DefaultStyle from './DefaultStyle';
+
+export interface DispatchProps {
+    onUpdateAuthor: (text: string) => void;
+    onUpdatePicture: (path: string) => void;
+}
+
+export interface StateProps {
+    author: Author;
+    navigation: any;
+}
+
+const tooltip = 'The name to author your posts';
+const namePlaceholder = 'Space Cowboy';
+
+export const IdentitySettings = (props: DispatchProps & StateProps) => {
+    return (
+        <KeyboardAvoidingView>
+            <Header onPressBack={() => { props.navigation.goBack(); }} />
+            <Text style={styles.tooltip}>{tooltip}</Text>
+            <SimpleTextInput
+                style={styles.row}
+                defaultValue={props.author.name}
+                onChangeText={props.onUpdateAuthor}
+                placeholder={namePlaceholder}
+                autoCapitalize='none'
+                autoFocus={props.author.name === ''}
+                autoCorrect={false}
+            />
+            <Text style={styles.tooltip}>Avatar</Text>
+            <View style={styles.imagePicker}>
+                {props.author.faviconUri &&
+                <Image source={{uri: props.author.faviconUri}} style={DefaultStyle.favicon} />}
+                <Button
+                    title='Pick Image'
+                    onPress={async () => {
+                        await openImagePicker(props.onUpdatePicture);
+                    }}
+                />
+            </View>
+        </KeyboardAvoidingView>
+    );
+};
+
+const openImagePicker = async (onUpdatePicture: (path: string) => void) => {
+    const imageData = await AsyncImagePicker.launchImageLibrary();
+    if (imageData != null) {
+        onUpdatePicture(imageData.uri);
+    }
+};
+
+const Header = (props: { onPressBack: () => void }): React.ReactElement<{}> => {
+    return (
+        <View style={styles.headerContainer}>
+            <View style={{ paddingLeft: 15 }}>
+                <TouchableOpacity
+                    onPress={props.onPressBack}
+                    activeOpacity={1.0}
+                    hitSlop={{
```

Comment:
You can also use TouchableView here which has a default hitSlop of 30

Reply:
fixed in 9be93f9328558afb5e61d224446f772a29cb0e32